storage/memory/memory_test.go

```diff
@@ -281,10 +301,48 @@ func TestObjects(t *testing.T) {
 		}
 	}
 	if cnt != 3 {
-		t.Errorf("g.Objects(%s, %s) failed to retrieve 3 objects, got %d instead", ts[0].Subject(), ts[0].Predicate(), cnt)
+		t.Errorf("g.Objects(%s, %s) failed to retrieve 3 objects, got %d instead ", ts[0].Subject(), ts[0].Predicate(), cnt)
 	}
 }
 
+func TestObjectsOffset(t *testing.T) {
+	ts, ctx := getTestOffsetTriples(t), context.Background()
+	g, _ := NewStore().NewGraph(ctx, "test")
```

Comment:
Can you handle the error here?
Checking if it's nil, and if not, print an error.

Reply:
Done